### PR TITLE
[148] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by DevOge**
Implementation Pictures.

IMG-6857

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
You have previously created a product page file, you dont have to create another one. You should only add the links to the pages.

**Comment by DevOge** [comment]
> You have previously created a product page file, you dont have to create another one. You should only add the links to the pages.

I didn't create a new file I only copied it into the htmlforms folder so it would work, the links were not working otherwise... should I take it out ma?

File: submissions/OgeEjim/html-forms/ContactForm.html [hunk 1]
```diff
@@ -5,6 +5,10 @@
    <title>Enquires</title>
 </head>
 <body>
+    <div>
+        <a href="ContactForm.html">Contact Form</a>
+        <a href="productpageforcinnamoncakes.html">Product</a>
```
Comment (adaugochi) [inline submissions/OgeEjim/html-forms/ContactForm.html:10]: This is wrongly linked, the `productpageforcinnamoncakes.html` is inside your `HTML` folder. doing it this `href="productpageforcinnamoncakes.html"` way won't target that file. So kindly correct yourself. Start from the folder name and go a step up

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/OgeEjim/html-forms/ContactForm.html (hunk 1 above)
Comment (adaugochi) [reply submissions/OgeEjim/html-forms/ContactForm.html:10]: You should not copy the `productpageforcinnamoncakes.html` from your `HTML` folder to `html-forms` folder

Comment (DevOge) [reply submissions/OgeEjim/html-forms/ContactForm.html]: > You should not copy the `productpageforcinnamoncakes.html` from your `HTML` folder to `html-forms` folder

good morning ma, I have made the corrections requested

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**